tests/stdlib/tslimsystem.nim

```diff
@@ -0,0 +1,7 @@
+discard """
+  nimout: '''
+tslimsystem.nim(7, 1) Warning: about to move out of system, import `std/syncio` instead; use `-d:nimPreviewSlimSystem` to enforce import; write is deprecated [Deprecated]
+'''
+"""
+
+write(stdout, "hello")
```

Comment:
This also gives a `stdout is deprecated` warning without the deprecation message (#9331). It would be great if #20115 is merged which fixes #9331 so a full deprecation message is given.